README.md

```diff
@@ -13,66 +13,65 @@ We try to increase our contribution to the iOS developer community through:
 - encouraging our team members to join iOS related meetups, groups and conferences
 - encouraging our team members to speak at iOS events
 - contributing back to some of the other open source libraries we use through issues and PRs
-- encouraging our team members to write tech articles about any topic they desire on our [engineering blog](https://engineering.nodesagency.com/)
+- encouraging our team members to write tech articles about any topic they desire on our [engineering blog](https://engineering.nodesagency.com/); if needed, we can also suggest topics for new articles
 
 We create lots of apps for our clients. Most of those apps rely heavily on web APIs. We abstracted the common operations for working with web APIs, and that's how [Serpent](https://github.com/nodes-ios/Serpent) (formerly known as Serializable) and [Cashier](https://github.com/nodes-ios/Cashier) were born. Another issue that was common in all the apps we do was handling keyboard appearance events. This led to the appearance of [KeyboardHelper](https://github.com/nodes-ios/KeyboardHelper).
```

Comment:
Maybe this should not mention all the repo's and isntead point to an org with all the OSS frameworks? :)

Reply:
It's not all the repos, there are way more on `nodes-ios` :). I think it gives a nice story of how some of our most used libraries came to be. If you want, I can remove it and just say something like 

We also make open source stuff sometimes. You can find it [here](https://github.com/nodes-ios/).

But I think that sounds a bit dull.

Reply:
Fair, leave it there :) It’s also not my decision to make on this, better to ask @chriscombs  😇